### ISSUE TITLE
Add support in the parser (D.S.L)

### ISSUE THREAD

**Issue by wirednkod** [paritytech/zombienet#1307]

**Comment by wirednkod** [comment]
@pepoviola  can you elaborate a bit more? Why is there a need to add support of `delay` on the parser, since it is supported from the `toml` file?

**Comment by pepoviola** [comment]
> @pepoviola can you elaborate a bit more? Why is there a need to add support of `delay` on the parser, since it is supported from the `toml` file?

Yes, because currently we can __only__ set the delay settings as part of the spawning phase. So, ones the network is running you __can't__ modify any of those settings. To allow to add/rm/modify the running network we should support modify the deployed manifest through our D.S.L, so users can for example increase or decrease the latency as part of their test. Make sense?
Thanks!
